pkg/osbuild/source.go

```diff
@@ -13,12 +13,13 @@ import (
 )
 
 // RpmDownloader specifies what backend to use for rpm downloads
-// Note that the librepo backend requires a newer osbuild.
+// Note that the librepo backend requires osbuild v138 from
+// 2025-01-15.
 type RpmDownloader uint64
 
 const (
-	RpmDownloaderCurl    = iota
-	RpmDownloaderLibrepo = iota
+	RpmDownloaderCurl = iota
```

Comment:
I do wonder if we should flip the order around and make librepo default now? The downside would be that we would potentially start generate manifests that are incompatible if we have systems that have a newer "images" library but an osbuild release that is older than a year.